src/index.html

```diff
@@ -5,8 +5,337 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="img">
+        <img
+          class="logo"
```

Comment:
```suggestion
      <a href="#home" class="img">
        <img
          class="img__logo"
```

if you create a separate bem block, `logo` should be an element of this